tenseal/__init__.py

```diff
@@ -1,7 +1,11 @@
 """TenSEAL is a library for doing homomorphic encryption operation on tensors.
 """
 
-import _tenseal_cpp as _ts_cpp
+try:
+    import _tenseal_cpp as _ts_cpp
+except ImportError:
+    import tenseal._tenseal_cpp as _ts_cpp
+
```

Comment:
So bazel put it inside the python package, actually this would be even cool to have without bazel, I just didn't know how to do it with setuptools